ebos/ecloutputblackoilmodule.hh

```diff
@@ -685,6 +688,98 @@ public:
                         = tracerModel.tracerConcentration(tracerIdx, globalDofIdx);
                 }
             }
+
+            // flows
+            if (!problem.model().linearizer().getFlowsInfo().empty()){
+                const auto& flowsInf = problem.model().linearizer().getFlowsInfo();
+                auto flowsInfos = flowsInf[globalDofIdx];
+                for (auto& flowsInfo : flowsInfos) {
+                    if (flowsInfo.faceId == 1){
+                        if (FluidSystem::phaseIsActive(gasPhaseIdx) & !this->flowsi_[gasCompIdx].empty())
```

Comment:
The `&` should probably be a `&&`. That is logical and, not bitwise and. Maybe one of the most easy to misuse operators in C++! Using it indicates that you are doing bit manipulation, and not pure boolean ops.